### ISSUE TITLE
Dashboard: Screen Reader A11y updates

### ISSUE THREAD

**Issue by carloskelly13** [GoogleForCreators/web-stories-wp#4683]
## Feature Description

1. Scroll To Top Button on all story views: When clicked or entered, should focus shift to top of view;
2. Publisher Logo Group on the Settings view reads out all stories as "Currently Selected" when group is focused.
3. Publisher Logo Group on the Settings view; Default logo's 'make default' option is disabled but does not announce as unavailable.
4. In the left navigation panel the current view should be read out by screen reader to give context to user.
5. The close button in the template details view should give more context. Since it is not a dialog it should tell the user where they are navigating to when they close it.
6. Telemetry Checkbox on the Settings View; On check exits page and enters it again, focus on this checkbox is weird.

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by BrittanyIRL** [comment]
1:  [Scroll to top](https://github.com/google/web-stories-wp/pull/4697)
2, 3, 6 : [Editor settings updates](https://github.com/google/web-stories-wp/pull/4695)
4: [Side Nav](https://github.com/google/web-stories-wp/pull/4698)
5: [Template Details](https://github.com/google/web-stories-wp/pull/4699)

**Comment by BrittanyIRL** [comment]
There's one last bit of this that was left behind awaiting review that is now ready to be QA'd. Reopening to go through proper qa.

**Comment by BrittanyIRL** [comment]
@csossi for QA on this (why i reopened) see this ticket: https://github.com/google/web-stories-wp/pull/4695 

```
Publisher Logo Group on the Settings view reads out all stories as "Currently Selected" when group is focused.
Publisher Logo Group on the Settings view; Default logo's 'make default' option is disabled but does not announce as unavailable.
Telemetry Checkbox on the Settings View; On check exits page and enters it again, focus on this checkbox is weird.
```

User facing changes: 
Navigate to dashboard -> editor settings:
- when focused on uploaded logo group, sr will read out 'Viewing existing publisher logos'
- When focused on individual logos in uploaded logo group, sr will read out 'Publisher logo [title of image uploaded]'
- When focused on telemetry checkbox and sr is reading notice, when it gets to the link it will read out 'Learn more by visiting google privacy policy'
- when a popover menu is expanded on an uploaded logo and the 'set as default' option is disabled (if the logo is already the default), sr will announce '[label] (currently unavailable)'
- when using a keyboard to check or uncheck telemetry checkbox, focus will return to checkbox after interaction

**Comment by csossi** [comment]
Verified in QA
